lib/internal/heap_utils.js

```diff
@@ -1,16 +1,37 @@
 'use strict';
 const {
-  Symbol
+  Symbol,
+  Uint8Array,
 } = primordials;
 const {
   kUpdateTimer,
   onStreamRead,
 } = require('internal/stream_base_commons');
 const { owner_symbol } = require('internal/async_hooks').symbols;
 const { Readable } = require('stream');
+const { validateObject, validateBoolean } = require('internal/validators');
 
 const kHandle = Symbol('kHandle');
 
+function getHeapSnapshotOptions(options = {
+  exposeInternals: false,
+  exposeNumericValues: false
+}) {
+  validateObject(options, 'options');
+  if (options.exposeInternals === undefined) {
+    options.exposeInternals = false;
+  }
+  if (options.exposeNumericValues === undefined) {
+    options.exposeNumericValues = false;
+  }
```

Comment:
For these I wonder if treating `undefined` and `null` the same would be fine?
 
```suggestion
  options.exposeInternals ??= false;
  options.exposeNumbericValues ??= false;
```